terminus

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * If we're running from phar load the phar autoload file.
+ */
+$pharPath = \Phar::running(true);
```

Comment:
If your .phar is not built correctly, \Phar::running() will always return an empty string, and this generic boilerplate will fail when run as a .phar.  If you build your .phar with [box2](https://github.com/box-project/box2), though, this is not an issue.  I would recommend replacing the Terminus phar scripts with a box2.json file in any event.